src/NodaTime.Web/Markdown/developer/website.md

```diff
@@ -44,17 +40,8 @@ directory containing DocFX outputs for each NodaTime version, for the API
 documentation.
 
 This directory is `src/NodaTime.Web/docfx/`, and can be created empty (no
-API documentation will be available), or can be populated with real outputs.
-
-The
-[`build/buildapidocs.sh`](https://github.com/nodatime/nodatime/blob/master/build/buildapidocs.sh)
-script can be run to generate the content (into `build/tmp/docfx/`) from the
-published NuGet packages.
-
-Alternatively, for local use, the
-[`build/clonedocfx.sh`](https://github.com/nodatime/nodatime/blob/master/build/clonedocfx.sh)
```

Comment:
This doesn't appear to exist any more. I don't know if there's an equivalent to fetch from the `history` branch?